app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,23 @@
+class OrdersController < ApplicationController
+  def new
+    @facade = Orders::NewFacade.new
+  end
+
+  def create
+    @order = Order.new order_params
+    @order.user_id = current_user.id
```

Comment:
move all business logic to `operations/orders/create.rb` => `Orders::Create`

*Hint:* `order_params.merge(user_id: user.id)` in those operation will give the necessary params with user included